Task/python/Taskfile.yaml

```diff
@@ -0,0 +1,53 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  IMAGE_NAME: null
+  VERSION: null
+  PYTHON_VERSION: null
+
+tasks:
+  lint:
+    preconditions:
+      - sh: test `git ls-files -z -o --exclude-standard | wc -w` = "0"
```

Comment:
What's the benefit of this over:
> test `git status -s | wc -w` = "0"
?

Reply:
I agree here, that should be the final version. The current usage is really the end result of trying a few different approaches that were a bit more explicit but they didn't work as well as I would have liked.